llvm/lib/Target/AMDGPU/SILoadStoreOptimizer.cpp

```diff
@@ -61,6 +61,7 @@
 #include "AMDGPU.h"
 #include "GCNSubtarget.h"
 #include "MCTargetDesc/AMDGPUMCTargetDesc.h"
+#include "SIDefines.h"
```

Comment:
Honestly, I'd make several groups here and then leave blank lines to prevent clang-format from complaining about it but it's completely up to you.

```
// This is always the first one
#include "SILoadStoreOptimizer.h"

#include "AMDGPU.h"
#include "GCNSubtarget.h"
#include "SIDefines.h"

 #include "MCTargetDesc/AMDGPUMCTargetDesc.h"

// then llvm headers
```